ditto/src/main/java/eu/xenit/testing/ditto/api/AlfrescoBootstrapper.java

```diff
@@ -1,56 +1,70 @@
-                userStoreContainer.type(System.CONTAINER);
-                userStoreContainer.storeRefIdentifier("alfrescoUserStore");
-                userStoreContainer.storeRefProtocol("user");
+            Node userStoreContainer = txn.addNode(userStoreRoot, System.CHILDREN, node -> {
+                node.type(System.CONTAINER);
+                node.storeRefIdentifier("alfrescoUserStore");
+                node.storeRefProtocol("user");
             });
 
-            txn.addNode(people -> {
-                people.type(System.CONTAINER);
-                people.storeRefIdentifier("alfrescoUserStore");
-                people.storeRefProtocol("user");
+            Node people = txn.addNode(userStoreContainer, System.CHILDREN, node -> {
+                node.type(System.CONTAINER);
+                node.storeRefIdentifier("alfrescoUserStore");
+                node.storeRefProtocol("user");
             });
 
-            txn.addNode(admin -> {
-                admin.type(User.USER);
-                admin.property(User.USERNAME, "admin");
+            txn.addNode(people, System.CHILDREN, node -> {
+                node.type(User.USER);
+                node.property(User.USERNAME, "admin");
             });
         });
 
         builder.skipToTransaction(6)
-                .addTransaction(txn -> txn
-                        .skipToNodeId(12)
-                        .addNode(node -> node
-                                .type(System.STORE_ROOT)
-                        )
-                        .addFolder("Company Home", node -> {
-                            node.mlProperty(Content.TITLE, "Company Home");
-                            node.mlProperty(Content.DESCRIPTION, "The company root space");
-                        })
-                        .addFolder("Space Templates")
-
-                        .skipToNodeId(26)
-                        .addFolder("User Homes")
-                );
+                .addTransaction(txn -> {
+                    txn.skipToNodeId(12);
+                    Node root = txn.addRoot(node -> {
+                    });
+                    Node companyHome = txn.addNode(root, System.CHILDREN, node -> {
+                        node.type(Content.FOLDER);
+                        node.name("Company Home");
+                        node.qname(Application.createQName("company_home"));
+
+                        node.aspect(Content.TITLED);
+                        node.mlProperty(Content.TITLE, "Company Home");
+                        node.mlProperty(Content.DESCRIPTION, "The company root space");
+
+                        node.aspect(Application.UIFACETS);
+                        node.property(Application.ICON, "space-icon-default");
+                    });
+                    Node dataDictionary = txn.addNode(companyHome, Content.CONTAINS, node -> {
+                        node.name("Data Dictionary");
+                    });
+                    txn.addNode(dataDictionary, node -> {
```

Comment:
no AssocType needed here?

Reply:
defaults to `cm:contains` now - I'll simplify the line above :+1: